src/exec/processor.rs

```diff
@@ -103,8 +103,8 @@ impl MapCompiler {
 
     let const_exprs = izip!(0..exprs.len(), exprs).filter(|e| match *e.1.kind() {ExprKind::Const(_) => true, _ => false});
     let nonconst_exprs = izip!(0..exprs.len(), exprs).filter(|e| match *e.1.kind() {ExprKind::Const(_) => false, _ => true});
-    
-    let schema_map = schema.to_map();    
+
```

Comment:
Right. These spaces were automatically removed by Atom. 
In addition, I'm quite confused on the location of the left parenthesis. In some lines, the left parentheses locate at the end of the line, while others locate at the start of the new line.
Do you have any preference on coding convention?